guides/micronaut-openapi-generator-client/java/src/test/java/example/twitter/api/TweetsApiTest.java

```diff
@@ -0,0 +1,159 @@
+// tag::imports[]
+package example.twitter.api;
+
+import example.twitter.model.SearchCount;
+import example.twitter.model.Tweet;
+import example.twitter.model.TweetCountsResponse;
+import example.twitter.model.TweetSearchResponse;
+import example.twitter.model.User;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+// end::imports[]
+
+//tag::class-begin[]
+/**
+ * API tests for TweetsApi
+ */
+@MicronautTest // <1>
```

Comment:
Disable the tests if environment variables are not present since this test requires them.

I think the following should work but please test that the tests execute if you set the environment variable. 

```
@EnabledIfEnvironmentVariable(named = "TWITTER_AUTH_CLIENT_ID", matches = ".*")
@EnabledIfEnvironmentVariable(named = "TWITTER_AUTH_CLIENT_SECRET", matches = ".*")
```